.github/env/Linux/gradle.properties

```diff
@@ -1,6 +1,13 @@
+# TODO (deephaven-core#639): Customize gradle settings per-CI job
```

Comment:
https://github.com/deephaven/deephaven-core/issues/87 , I'm not sure exactly what @mofojed means when he says permalink will be auto added?

Reply:
We don't have this documented in our `CONTRIBUTING.md` guide or in our wiki anywhere... @chipkent to decide if it should be documented anywhere.
When discussing previously, we had settled on just the issue number if in the same repo, or repo and issue number if in a different repo. In this case, it would be `# TODO: #639 ...`. These comments are on issue #87 . 
Also the GitHub plugin for VSCode has a handy trigger that will automatically create issues for you, and link them back to the code: https://code.visualstudio.com/docs/editor/github#_creating-issues
I'm not seeing a similar plugin for IntelliJ.